src/main/java/com/internet/shop/dao/jdbc/ProductDaoJdbcImpl.java

```diff
@@ -0,0 +1,109 @@
+package com.internet.shop.dao.jdbc;
+
+import com.internet.shop.dao.interfaces.ProductDao;
+import com.internet.shop.exeptions.DataProcessingException;
+import com.internet.shop.library.Dao;
+import com.internet.shop.model.Product;
+import com.internet.shop.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class ProductDaoJdbcImpl implements ProductDao {
+    @Override
+    public Product create(Product product) {
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement statement = connection
+                    .prepareStatement(
```

Comment:
```suggestion
            PreparedStatement statement = connection.prepareStatement(
```
don't make code longer unnecessary

Reply:
done